spec/javascripts/health-care/components/form-elements/ErrorableCheckbox.spec.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import SkinDeep from 'skin-deep';
+
+import ErrorableCheckbox from '../../../../../_health-care/_js/components/form-elements/ErrorableCheckbox';
+
+describe('<ErrorableCheckbox>', () => {
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    it('label is required', () => {
+      SkinDeep.shallowRender(
+        <ErrorableCheckbox onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `label` was not specified in `ErrorableCheckbox`/);
+    });
+
+    it('label must be a string', () => {
+      SkinDeep.shallowRender(
+        <ErrorableCheckbox label onValueChange={(_update) => {}}/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `label` of type `boolean` supplied to `ErrorableCheckbox`, expected `string`./);
```

Comment:
Removing ending period from regexp. It's not in the other propType tests and doesn't quite do the right thing anyways.